build/configs/imxrt1050-evk/tc-protected/defconfig

```diff
@@ -175,12 +177,15 @@ CONFIG_IMXRT_LPSPI1=y
 CONFIG_IMXRT_I2S=y
 CONFIG_IMXRT_I2S_TX=y
 CONFIG_IMXRT_I2S_RX=y
-# CONFIG_IMXRT_SEMC is not set
+CONFIG_IMXRT_SEMC=y
 # CONFIG_IMXRT_SNVS_LPSRTC is not set
 # CONFIG_IMXRT_SNVS_HPRTC is not set
 # CONFIG_IMXRT_USDHC is not set
 # CONFIG_IMXRT_GPIO_IRQ is not set
 
+CONFIG_IMXRT_SEMC_SDRAM=y
+CONFIG_IMXRT_SDRAM_START=0x80000000
+CONFIG_IMXRT_SDRAM_SIZE=33554432
 #
 # Memory Configuration
 #
```

Comment:
1. You should change `CONFIG_IMXRT_OCRAM_PRIMARY` to `CONFIG_IMXRT_SDRAM_PRIMARY`.
2. Please don't modify this manually. You should modify this through menuconfig and then update this by copying .config.

Reply:
updated